src/Components/WebAssembly/WebAssembly/src/Infrastructure/JSInteropMethods.cs

```diff
@@ -34,23 +35,38 @@ public static void NotifyLocationChanged(string uri, bool isInterceptedLink)
         /// For framework use only.
         /// </summary>
         [JSInvokable(nameof(DispatchEvent))]
-        public static Task DispatchEvent(WebEventDescriptor eventDescriptor, string eventArgsJson)
+        public static async Task DispatchEvent(string eventDescriptorJson, string eventArgsJson)
```

Comment:
I think @SteveSandersonMS did this, but with Tanay's work, this can be a `byte []`